Add osu! Philippines: CONQuest PH 2022 Newspost (#7671)
ppy/osu-wiki: news/2022-07-21-osu-philippines-conquest-ph-2022.md

---
layout: post
title: "osu! Philippines: CONQuest PH 2022"
date: 2022-07-21 00:00:00 +0000
---

> A monumental first for osu! in the Philippines, an osu!booth will be open at [CONQuest PH 2022](https://conquestph.com), held at the SMX Convention Centre, Pasay City, Philippines.

Reviewer: ```suggestion
A monumental first for osu! community in the Philippines, an (unofficial) osu! community run space will be open at [CONQuest PH 2022](https://conquestph.com), held at the SMX Convention Centre, Pasay City, Philippines.
```

> The first ever LAN event, the **Quickfire Showdown**, will be hosted at the booth! 

Reviewer 2: ```suggestion
The first ever Filipino LAN event, the **Quickfire Showdown**, will be hosted at the booth! 
```

more correct this way i guess?

![Quickfire Showdown Banner](/wiki/shared/news/2022-07-21-osu-philippines-conquest-ph-2022/quickfire_showdown_banner.png) 

As an extra special addition, [BTMC](https://osu.ppy.sh/users/3171691) will be present at the event playing with the talented players of the osu!Philippines community! 

## Tournament

The Quickfire Showdown, the main event of CONQuest PH 2022, is an open tournament held on both days for those present at the venue. As the name implies, there will be no warmups, and the mappools will be revealed on the spot. 

Additionally, as a CONQuest PH 2022 exclusive, there will be a track from one of osu!'s featured artists, [LEAF XCEED Music Division](https://osu.ppy.sh/beatmaps/artists/88)! 

## Prizes

### July 23, 2022 (Day 1)

| Placing | Prizes |
| :-: | :-- |

> | ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | Profile badge + osu!merch + 3 months of osu!supporter |

Reviewer 2: ```suggestion
| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, osu!merch, 3 months of osu!supporter |
```

> | Runner-up | osu!merch |

Reviewer 2: might wanna use the silver crown for runner-up?

Author: looks good, sure!

### July 24, 2022 (Day 2)

| Placing | Prizes |
| :-: | :-- |

> | ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | Profile badge + osu!merch + 3 months of osu!supporter |

Reviewer 2: ```suggestion
| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, osu!merch, 3 months of osu!supporter |
```

| Runner-up | osu!merch |

## Schedules

### July 23, 2022 (Day 1)

| Event | Timestamp |
| :-: | :-- |
| Registration phase | 10:00 - 13:00 UTC+8 |
| Round of 16 | 13:00 - 15:00 UTC+8 |
| Quarterfinals | 15:00 - 16:00 UTC+8 |
| Semifinals | 16:00 - 17:00 UTC+8 |
| Finals | 17:00 UTC+8 |

### July 24, 2022 (Day 2)

| Event | Timestamp |
| :-: | :-- |
| Registration phase | 10:00 - 13:00 UTC+8 |
| Round of 16 | 13:00 - 15:00 UTC+8 |
| Quarterfinals | 15:00 - 16:00 UTC+8 |
| Semifinals | 16:00 - 17:00 UTC+8 |
| Finals | 17:00 UTC+8 |

## Booth Activties

> Aside from the tournament, other activities will also be held at the booth, such as a player hub for fellow community members to meet up. A free play area with devices running osu!lazer will also be open for those both new and familiar with osu! to try out osu!lazer. osu!lazer will be preloaded with tracks from the [featured artists](https://osu.ppy.sh/beatmaps/artists). 

Reviewer 2: ```suggestion
Aside from the tournament, other activities will also be held at the booth, such as a player hub for fellow community members to meet up. A free play area with devices running osu!lazer will also be open for those both new and familiar with osu! to try out. osu!lazer will be preloaded with tracks from the [featured artists](https://osu.ppy.sh/beatmaps/artists). 
```

> Aside from the tournament, other activities will also be held at the booth, such as a player hub for fellow community members to meet up. A free play area with devices running osu!lazer will also be open for those both new and familiar with osu! to try out. osu!lazer will be preloaded with tracks from the [featured artists](https://osu.ppy.sh/beatmaps/artists). 

Reviewer: ```suggestion
Aside from the tournament, other activities will also be held at the booth, such as a player hub for fellow community members to meet up. A free play area with devices running osu!(lazer) will also be open for those both new and familiar with osu! to try out. 
```

Not sure why mentioning featured artists is required.

For our international audience or those who couldn't attend the convention, you can stay in the loop by watching the stream at [osu! Philippines Twitch Channel](https://twitch.tv/osuliveph).

> See you in-stream and in-person!

Reviewer 2: ```suggestion
See you on-stream and in-person!
```

> For our international audience or those who couldn't attend the convention, you can stay in the loop by watching the stream at the [osu! Philippines Twitch channel](https://twitch.tv/osuliveph).

Reviewer: ```suggestion
For those who can't attend the convention but are interested in following along, you can stay in the loop by watching live-streamed content at the [osu! Philippines Twitch channel](https://twitch.tv/osuliveph).
```
